tests/MongoDBDriverReadPreference-001.phpt

```diff
@@ -31,7 +33,9 @@ foreach ($data as $item) {
 ?>
 --EXPECT--
 array (  0 => 1,):
+array (  0 => 1,  1 =>   array (  ),):
 array (  0 => 5,):
+array (  0 => 5,  1 =>   array (  ),):
```

Comment:
This may be redundant in light of the PHPC tests, but should you actually be `var_dump()`-ing the constructed ReadPreference here? This is only testing that the constructor did not throw an exception.